openerp/addons/base/static/src/css/description.css

```diff
@@ -698,9 +688,9 @@ div.oe_demo:hover span.oe_demo_play {
   background-image: +linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.02));
   box-shadow: 0px -3px 10px -5px rgba(0, 0, 0, 0.1) inset;
   cursor: pointer;
-  -webkit-transition: all 250ms linear;
   -moz-transition: all 250ms linear;
   -o-transition: all 250ms linear;
+  -webkit-transition: all 250ms linear;
   transition: all 250ms linear;
 }
 
```

Comment:
looks like there are useless modifications on this file (reordered lines, less browser specific rules).
maybe an issue with your sass compiler?